x-pack/plugins/apm/server/lib/status_check/server_check.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { PROCESSOR_EVENT } from '../../../common/constants';
+import { Setup } from '../helpers/setup_request';
+
+interface APMServerOnboardingDocument {
+  observer: {
+    listening: string;
+    hostname: string;
+    id: string;
+    ephemeral_id: string;
+    type: string;
+    version: string;
+    version_major: number;
+  };
+  '@timestamp': string;
+  ecs: {
+    version: string;
+  };
+  host: {
+    name: string;
+  };
+  processor: {
+    name: 'onboarding';
+    event: 'onboarding';
+  };
+}
```

Comment:
Can you define the onboarding document together with the other apm docs types?
https://github.com/elastic/kibana/tree/master/x-pack/plugins/apm/typings/es_schemas

Reply:
Btw. `es_schemas` should maybe be renamed to `apm_es_docs` (but that's not necessary for this PR)